src/compiler/crystal/tools/doc/macro.cr

```diff
@@ -39,7 +39,7 @@ class Crystal::Doc::Macro
   end
 
   def anchor
-    "#" + URI.escape(id)
+    '#' + URI.escape(id)
```

Comment:
This one I'm not sure sure : Char + String doesn't look quite good.

Reply:
Well, we should make it good if it's bad I guess.